app/views/orders/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class='lift-bacground'>
+
+<% content_for :title, I18n.t("orders.new.create_invoice") %>
+
+  <div class="wrapper">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12 col-sm-8 col-sm-offset-2">
+          <div class="dashboard-card">
+
+           <p class="text-center"> 
+              <%= t('.invoice_description', total_hours: (@minutes[:minutes_not_paid].to_f/60).round(1), total_min: @minutes[:minutes_not_paid] ) %> 
```

Comment:
* sounds like the controller method would be better placed in model
* in any case, `.to_f/60).round(1)` does not belong to views. Knowledge of conversions/rounding belongs to models, or separate classes (informally known as 'presenters').